src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -237,6 +239,19 @@ private static int GetCurrentProcessId()
             return Interop.Sys.GetPid();
         }
 
+        partial void ThrowIfExited(bool refresh)
+        {
+            if (_waitStateHolder == null && !refresh)
+            {
+                return;
+            }
+
+            if (GetWaitState().GetExited(out _, refresh))
+            {
+                throw new InvalidOperationException(SR.Format(SR.ProcessHasExited, _processId.ToString(CultureInfo.CurrentCulture)));
```

Comment:
Is the CultureInfo.CurrentCulture needed?

Reply:
I lifted this exception from `GetProcessHandle`. I don't know if it is needed.

Reply:
We can remove it.  It won't affect the default formatting.